internal/aws/cwlogs/cwlog_client.go

```
@@ -207,3 +210,9 @@ func matchContainerInsightsPattern(logGroupName string) bool {
 	r, _ := regexp.Compile(regexP)
 	return r.MatchString(logGroupName)
 }
+
+func matchEnhancedContainerInsightsPattern(logGroupName string) bool {
+	regexP := "^/aws/containerinsights/\\S+/performance$"
+	r, _ := regexp.Compile(regexP)
```

nit - should this be compiled once outside of the function call? Seems wasteful to constantly compile it, especially if we're just ignoring the err cause we know it's valid

---

Yeah, `regexp.MustCompile` as a global var.

---

sure, will update